index.html

```diff
@@ -2812,6 +2812,15 @@ <h2>Algorithms</h2>
 [=map=] with a `proof` [=map/key=].
       </p>
 
+      <p>
+In addition to the algorithms below, implementers MAY implement additional
+reasonable defaults and safeguards to mitigate things such as developer error,
```

Comment:
"things such as" could be improved.  Rewording would conflict with other suggestions here.  Perhaps something like "... mitigate developer error, [..., ...,] or other situations.".

Reply:
perhaps "...safeguards to mitigate things such as developer error..." => "...safeguards as mitigations for developer error..."

Reply:
@davidlehn @dlongley — See [my revised suggestion](https://github.com/w3c/vc-data-integrity/pull/276#discussion_r1661306150), above.